src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif, Arial;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 12px;
+}
+
+header {
+  margin: 10px 0 0;
```

Comment:
Remember the checklist item: Be consistent with your *vertical* margins. You've added a top margin to the `header` element here. It's generally a good practice to stick to either top or bottom margins for consistency and to avoid issues with margin collapsing.